std/mem.zig

```diff
@@ -515,11 +524,10 @@ pub fn writeInt(buf: []u8, value: var, endian: builtin.Endian) void {
 }
 
 pub fn writeIntBE(comptime T: type, buf: *[@sizeOf(T)]u8, value: T) void {
-    assert(T.bit_count % 8 == 0);
-    const uint = @IntType(false, T.bit_count);
-    if (uint == u0) {
-        return;
+    if (comptime T.bit_count % 8 != 0) {
```

Comment:
This `comptime` is superfluous.

Reply:
superfluous, meaning that that `comptime` is not required in the `if` statement?